src/internal/observable/dom/webSocket.ts

```diff
@@ -103,8 +103,8 @@ import { WebSocketSubject, WebSocketSubjectConfig } from './WebSocketSubject';
  * // Note that at least one consumer has to subscribe to the created subject - otherwise "nexted" values will be just buffered and not sent,
  * // since no connection was established!
  *
- * subject.next(JSON.stringify({message: 'some message'}));
- * // This will send a message to the server once a connection is made. Remember to serialize sent value first!
+ * subject.next({message: 'some message'});
+ * // This will send a message to the server once a connection is made. Remember value is by default serialized with JSON.stringify!
```

Comment:
```suggestion
 * // This will send a message to the server once a connection is made. Remember value is serialized with JSON.stringify by default!
```